scene/src/controllers/game.controller.ts

```diff
@@ -1,9 +1,120 @@
+import { getPlayerPosition } from '@dcl-sdk/utils'
+import { engine } from '@dcl/sdk/ecs'
+import type { Vector3 } from '@dcl/sdk/math'
+import {
+  loadPlaceFromApi,
+  loadSceneFromBevyApi,
+  savePlayerPosition
+} from 'src/state/sceneInfo/actions'
+import { store } from 'src/state/store'
+import {
+  fetchPlaceFromCoords,
+  getPlaceFromApi,
+  updateFavoriteStatus,
+  updateLikeStatus
+} from 'src/utils/promise-utils'
+import { type ReadOnlyVector3 } from '~system/EngineApi'
 import { UIController } from './ui.controller'
+import type { LiveSceneInfo } from 'src/bevy-api/interface'
+import { BevyApi } from 'src/bevy-api'
 
 export class GameController {
+  public timer: number = 0
   uiController: UIController
   constructor() {
     this.uiController = new UIController(this)
+    engine.addSystem(this.positionSystem.bind(this))
+    this.restartTimer()
+    // void this.getFavorites()
+  }
+
+  // public async getFavorites(): Promise<void> {
+  //   const favoritePlaces = await getFavoritesFromApi()
+  //   store.dispatch(loadFavoritesFromApi(favoritePlaces))
+  // }
+
+  positionSystem(): void {
+    if (engine.PlayerEntity !== undefined) {
+      const newPosition: Vector3 = getPlayerPosition()
+      const oldPosition: Vector3 | undefined =
+        store.getState().scene.explorerPlayerPosition
+
+      if (
+        oldPosition === undefined ||
+        Math.floor(newPosition.x / 16) !== oldPosition.x ||
+        Math.floor(newPosition.z / 16) !== oldPosition.z
+      ) {
+        store.dispatch(
+          savePlayerPosition({
+            x: Math.floor(newPosition.x / 16),
+            y: 0,
+            z: Math.floor(newPosition.z / 16)
+          })
+        )
+        void this.updateWidgetPlace()
```

Comment:
```suggestion
        this.updateWidgetPlace().catch(console.error)
```

please, replace all calls that has "void call()" for ignoring the promise, to call().catch(console.error)